rudder-core/src/main/scala/com/normation/rudder/services/policies/InterpolatedValueCompiler.scala

```diff
@@ -59,19 +61,26 @@ import com.normation.inventory.domain.NodeInventory
  * ${rudder.xxx}
  * were "xxx" is the parameter to lookup.
  *
- * We handle 2 kinds of parameterizations:
+ * We handle 3 kinds of parameterizations:
  * 1/ ${rudder.param.XXX}
  *    where:
  *    - XXX is a parameter configured in Rudder
  *      (for now global, but support for node-contextualised is implemented)
  *    - XXX is case sensisite
  *    - XXX's value can contains other interpolation
+ *
  * 2/ ${rudder.node.ACCESSOR}
  *    where:
  *    - "node" is a keyword ;
  *    - ACCESSOR is an accessor for that node, explained below.
  *    - the value can not contains other interpolation
  *
+ * 3/ ${node.properties[keyone][keytwo]
+ *     where:
+ *       - keyone, keytwo are path on json values
+ *      - the return value is the string representation, in compact mode, of the resulting access
+ *      - if the key is not found, we raise an error.
+ *
```

Comment:
Comments should be updated to state that spaces are authorized in parameters, so that it won't be surprising